consensus/Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "bc"
 [dependencies]
 amplify = { workspace = true }
 strict_encoding = { workspace = true }
-strict_types = { workspace = true, optional = true }
+strict_types = { workspace = true }
```

Comment:
No reason in doing it non-optional

Reply:
This is not optional anymore because of implementation of StrictSerialize and StrictDeserialize for CompressedPk, anyway bp-consensus depends on commit_verify which depends on strict_types, so nothing actually changed